giskard/rag/evaluate.py

```diff
@@ -15,6 +16,9 @@
 
 logger = logging.getLogger(__name__)
 
+REPORT_CORRECTNESS_COL = "correctness"
```

Comment:
from what I understand this depends on `CorrectnessMetric::name`, so does not belong here